src/feature/nodelist/routerlist.c

```diff
@@ -520,6 +520,7 @@ router_add_running_nodes_to_smartlist(smartlist_t *sl, int need_uptime,
                                       int need_capacity, int need_guard,
                                       int need_desc, int pref_addr,
                                       int direct_conn,
+                                      bool rendezvous_v3,
```

Comment:
The number of arguments to this function are getting out of hand.  Can we have a ticket to use flags or a struct instead?

Reply:
Ah never mind, it looks like you did that in a later commit.